nft/v3/jobs.go

```diff
@@ -124,21 +125,20 @@ func (c *CommitAndMintNFTJobRunner) New() gocelery.Runner {
 	mj := &CommitAndMintNFTJobRunner{
```

Comment:
So this runner function will now take care of minting the first time a document is committed, from a state of pending. I find the name a bit confusing, knowing that the other runner `MintNFTJobRunner` will only work once the document has been committed previously so essentially making it an update document call. Maybe we can be more explicit on the naming of each, like:
`MintNFTJobRunner ` -> `UpdateMintNFTJobRunner`
or
`CommitAndMintNFTJobRunner ` -> `PendingMintNFTJobRunner`